src/System.Transactions/src/System/Transactions/SafeIUnknown.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+namespace System.Transactions
+{
+    // Keep an interface pointer that will not be used in a SafeHandle derived so
+    // that it will be properly released.
+    internal sealed class SafeIUnknown : SafeHandle
+    {
+        private SafeIUnknown() : base(IntPtr.Zero, true) { }
```

Comment:
>private [](start = 8, length = 7)

Why did you make this private? In the desktop code, this is internal, just like the other constructor.